python/pyarrow/tests/test_pandas.py

```diff
@@ -4513,3 +4513,27 @@ def test_does_not_mutate_timedelta_nested():
     df = table.to_pandas()
 
     assert df["timedelta_2"][0].to_pytimedelta() == timedelta_2[0]
+
+
+def test_list_no_duplicate_base():
```

Comment:
There is a `TestConvertListTypes` class that groups some list type related tests, maybe can move this somewhere there.

Reply:
I can move it there.